net-dialup/picocom/picocom-3.1.ebuild

```diff
@@ -1,18 +1,17 @@
-# Copyright 1999-2023 Gentoo Authors
+# Copyright 1999-2024 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 EAPI=8
 
 inherit toolchain-funcs
 
-DESCRIPTION="minimal dumb-terminal emulation program"
+DESCRIPTION="A minimal dumb-terminal emulation program"
```

Comment:
Not quite what I requested, I usually try to avoid filler words at all cost. Works either way though.